pkg/controller/migmigration/task.go

```diff
@@ -167,14 +170,15 @@ var StageItinerary = Itinerary{
 		{Name: WaitForVeleroReady, Step: StepStageBackup},
 		{Name: WaitForResticReady, Step: StepStageBackup, all: HasPVs | HasStagePods},
 		{Name: WaitForRegistriesReady, Step: StepStageBackup, all: IndirectImage | EnableImage | HasISs},
-		{Name: EnsureCloudSecretPropagated, Step: StepStageBackup},
+		{Name: EnsureCloudSecretPropagated, Step: StepStageBackup, any: IndirectImage | IndirectVolume},
```

Comment:
We need to be careful here. This is needed any time there's a velero restore, which means any time there's a velero backup. For final itnerary, we always have this (but that's not affected here, so that's fine). For stage itineary, maybe what we really want here is `all: HasStageBackup` instead of the indirect image/volume check?